### ISSUE TITLE
npm publish fails to update published package: Failed PUT 404

### ISSUE THREAD

**Issue by mihaildu** [npm/npm#21038]
#### I'm opening this issue because:

  - [x] npm is producing incorrect or undesirable behavior.

#### What's going wrong?

I published a package with `npm publish --access public` under `@elifesciences` org. The package can be found here https://www.npmjs.com/package/@elifesciences/elife-theme

After I updated the files for this package, I did (while logged in as an user with admin rights)
`npm version patch`
`npm publish`

And now I get
```
npm ERR! publish Failed PUT 404
npm ERR! code E404
npm ERR! 404 Not found : @elifesciences/elife-theme
npm ERR! 404 
npm ERR! 404  '@elifesciences/elife-theme' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
```
and in the log file
https://gist.github.com/dryhten/f0cd7ea4cb0a2fee46ba50d55302f08e

### supporting information:

 - `npm -v` prints: 6.1.0
 - `node -v` prints: v8.11.2
 - `npm config get registry` prints: https://registry.npmjs.org/
 - Windows, OS X/macOS, or Linux?: Debian GNU/Linux 9.3 (stretch), Kernel 4.9.0-4-amd64 x86_64

**Comment by Viatorus** [comment]
I had the same issue for 8 days.

See the end of the log here:
https://travis-ci.org/LokiJS-Forge/LokiDB/builds/397214166

Two packages were published to npm but the third package failed. After doing it manually (today) it worked with the same command...

**Comment by mihaildu** [comment]
So I started looking into this issue and managed to solve it. I do not own this npm org, but I am part of the team that owns the package and I have the admin role. However, this team had only read rights set for the package. So what I did was the following:

Go to Profile settings on npmjs.com > Organizations (on the left side) > select the npm org that has the package > Teams tab (green one at the top) > Packages > click on read/write for the package (instead of read). Then I did `npm publish` and it worked.

I guess a neat feature would be to see the rights you have for a package rather than looking at the team :)
